live-examples/js-examples/string/string-iterator-foreach.html

```diff
@@ -0,0 +1,8 @@
+<pre>
+<code id="static-js">var str = "The quick brown fox jumped over the lazy dog's back."
```

Comment:
I think, since these show an ES6 feature (I think...) we should be able to use `const ` and `let` here, as per the style guide (https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#language-choice-es5es6).

Here `const` seems like a good choice.

Also, please use single quotes for strings: https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#single-quotes.